### DISCUSSION TITLE
Question about username

### DISCUSSION THREAD

**Discussion by LukasHaemmerli** [Louisvdw/dbus-serialbattery#223] [Q&A] [answered]
hi theres something i don't understand:

after intsalling the driver, if i would to chek it i get this message:

Using username "root".
root@192.168.8.103's password:
Last login: Thu Sep 29 23:16:04 2022 from 192.168.8.157
**root@nanop**i:~# nano /data/etc/dbus-serialbattery/utils.py
root@nanopi:~# /data/log/dbus-serialbattery.ttyUSB0/current
-sh: /data/log/dbus-serialbattery.ttyUSB0/current: **No such file or directory**
root@nanopi:~#

do you know why the devices name changed to "nanopi"?

is it possible to use the daly instead of an smart shunt with a usb port with a usb hub (as i use the only port on my multi with the energy meter)?

**Comment by Louisvdw** [comment, marked as answer]
The build in GX devices (so for instance Multiplus II GX) use a NanoPi device inside. 
So NanoPi is just the name of the mini computer inside on which your VenusOS runs. You can find a list of the hardware mapped to the devices that Victron sells at https://github.com/victronenergy/venus/wiki/machines.

The built in GX devices do have less outside ports available, but if you add a USB-Hub and then connect all your USB devices to that it will work perfectly.

**Comment by LukasHaemmerli** [comment]
Hi Louis

 

Thank you for your fast answer, i guessed that with the name (but i’m an absolute newbie in that system)

Can i ask you more questions if i get some… ?

 

 

 

Oh btw how much is the coffee in your town?

 

Kind regards from the (not so sunny) switzerland

 

lukas

 

Von: Louis Van Der Walt ***@***.***> 
Gesendet: Freitag, 30. September 2022 15:24
An: Louisvdw/dbus-serialbattery ***@***.***>
Cc: LukasHaemmerli ***@***.***>; Author ***@***.***>
Betreff: Re: [Louisvdw/dbus-serialbattery] Question about username (Discussion #223)

 

The build in GX devices (so for instance Multiplus II GX) use a NanoPi device inside.
So NanoPi is just the name of the mini computer inside on which your VenusOS runs. You can find a list of the hardware mapped to the devices that Victron sells at https://github.com/victronenergy/venus/wiki/machines.

The built in GX devices do have less outside ports available, but if you add a USB-Hub and then connect all your USB devices to that it will work perfectly.

**Reply by Louisvdw** [reply]
Hi Lukas
The wiki and FAQ is a good source to help you get it installed. There is also a Youtube video in the how to install page that shows all the steps. 
You are always welcome to ask questions. I'll try to answer as best I can and it helps to know what is missing in the instructions. 

Coffee here in my town cost about ZAR 50 a cup, which would be around 3 CHF ☕.
I've been in Switzerland a few years ago. We took the Glacier Express to St Moritz and it was snowing most of the way. The next day on the Bernina Express was all sunshine 😃
